Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -105,6 +105,7 @@ protected function configure()
                     new InputOption('dry-run', '', InputOption::VALUE_NONE, 'Only shows which files would have been modified'),
                     new InputOption('level', '', InputOption::VALUE_REQUIRED, 'The level of fixes (can be psr0, psr1, psr2, or symfony (formerly all))', null),
                     new InputOption('using-cache', '', InputOption::VALUE_REQUIRED, 'Does cache should be used (can be yes or no)', null),
+                    new InputOption('cache-file', '', InputOption::VALUE_OPTIONAL, 'Cache file to be used', Config::CACHE_FILE),
```

Comment:
no, the `Config::CACHE_FILE` is default value for `Config` instance, not for CLI option. Leave option without default value